### PR TITLE
Added the first version of an abstract

### PR DESCRIPTION
**Pull request by m6a-UdS**
FIrst draft

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,6 @@
-# talko-lingo
+# Talko-Lingo
 A demonstration project for the re:Invent Builder's Fair
+
+## Abstract
+
+Talko-Lingo is a simple demonstration project that leverages several AWS technologies to enable multilingual communications. It presents itself as a twenty-first century "Talkie-Walkie" that can translate communications as they happen. Based on a simple Raspberry Pi, this open-source project will introduce attendees to how AWS cloud technologies used to bring life to Internet of Things projects, with a touch of 3D printing and electronics design. Attendees will also be able to try it by themselves and test the technology, with visual feedback and additional informations regarding the whole translation process.
```
Comment (laurrentt) [inline README.md]: You can multiline this using lines of 80 or 120 columns.

Comment (laurrentt) [reply README.md]: It's actually "Walkie-Talkie" and not "Talkie-Walkie".

File: README.md [hunk 2]
```diff
@@ -1,2 +1,8 @@
-# talko-lingo
+# Talko-Lingo
 A demonstration project for the re:Invent Builder's Fair
+
+## Abstract
+
+### Talko-Lingo: The multilingual Talkie-Walkie
+
+Talko-Lingo is a simple demonstration project that leverages several AWS technologies to enable multilingual communications. It presents itself as a twenty-first century "Talkie-Walkie" that can translate communications as they happen. Based on a simple Raspberry Pi, this open-source project will introduce attendees to how AWS cloud technologies used to bring life to Internet of Things projects, with a touch of 3D printing and electronics design. Attendees will also be able to try it by themselves and test the technology, with visual feedback and additional informations regarding the whole translation process.
```
Comment (spg) [inline README.md]: `to how AWS cloud technologies can be used to`

File: README.md (hunk 1 above)
Comment (laurrentt) [reply README.md]: I'm not sure about the "twenty-first century" thing. I think it's even more innovative than that. I would use "a modern twist", "Cloud powered", etc.

**Comment by m6a-UdS** [comment]
New propositions from Ralph (Will integrate in this PR):

## Abstract
### Talko-Lingo: The Multilingual Walkie Talkie
Talko-Lingo enables near real time communication between two people of the same or
different languages.  Users interface with a Walkie Talkie consisting of a Raspberry Pi
embedded into a 3D printed device.  After selecting their language each user can seamlessly
communicate with the recipient, who may be speaking a different language.  The project
leverages AWS cloud technologies to encode each users voice, transcribe it, translate it to the
target language and then finally convert the translated text to speech.  Built in display
mechanisms provide visual feedback of the translation process.

**Pull request merged**